### PR TITLE
feat: Add create new Trick Route

### LINKED ISSUE #296: Save personal tricks - Creation page

### PR DESCRIPTION
**Pull request by WaldemarLehner**
Closes #296

### CODE REVIEW

File: src/components/ui/form/GenericFormElement.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+const props = defineProps<{
+  id: string;
+  name: string;
+  description?: string;
+  issues?: string[];
+}>();
+
+const inputClasses = computed(() => {
+  const classes = `bg-gray-50 border  text-sm rounded-lg block w-full p-2.5`.split(' ');
```
Comment (bastislack) [inline src/components/ui/form/GenericFormElement.vue]: Between `border`and `text-sm` there are two spaces. Is that on purpose?

Comment (WaldemarLehner) [reply src/components/ui/form/GenericFormElement.vue]: fixed

**Comment by JulianDietzel** [comment]
Consider the small, but relevant update to the structure of `src/router/router.ts` done in #337. The change is merged an available on `rewrite`. You will have to merge `rewrite` and update your entry into `src/router/router.ts` accordingly.

**Comment by WaldemarLehner** [comment]
image

image


I would merge this *without* the VariantOf / recommend prereqs / videos Segment. These should be added in a separate PR to not bloat this PR even further.

**Pull request merged**